packages/opentelemetry-plugin-grpc/test/utils/SpanAuditProcessor.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Span } from '@opentelemetry/types';
+import { SpanAudit } from './SpanAudit';
+
+export class SpanAuditProcessor {
```

Comment:
Once #234 is merged, we should get ride of these temp helper classes.

Reply:
Refactored to use `InMemorySpanExporter` in 798bc4b